kolibri/core/assets/src/views/ContentIcon.vue

```diff
@@ -58,6 +58,13 @@
         :color="color"
         style="top:0;"
       />
+      <KIcon
+        v-if="is('bookmark')"
+        icon="bookmark"
+        :class="[colorClass]"
+        :color="color"
+        style="top:0;"
+      />
```

Comment:
Bookmarks are not "content" (in the sense of Kolibri resources that come from Studio channels), so they should not be added to `ContentIcon` or `ContentNodeKinds`.

Reply:
that sounds right to me

Reply:
I added the icon into new file `BookmarkIcon.vue`. Thanks